clients/drcachesim/drpt2trace/ir2trace.h

```diff
@@ -92,7 +92,7 @@ class ir2trace_t {
      * error code.
      */
     static ir2trace_convert_status_t
-    convert(DR_PARAM_IN drir_t &drir, DR_PARAM_INOUT std::vector<trace_entry_t> &trace,
+    convert(DR_PARAM_IN drir_t *drir, DR_PARAM_INOUT std::vector<trace_entry_t> &trace,
```

Comment:
If this pointer is DR_PARAM_IN, make it const.  If it gets modified it should not be _IN.

Reply:
I could've made it const but instrlist_first doesn't have a const param.

Reply:
It doesn't get modified so I'll let it be DR_PARAM_IN.